sdks/python/apache_beam/examples/cookbook/datastore_wordcount.py

```diff
@@ -15,7 +15,49 @@
 # limitations under the License.
 #
 
-"""A word-counting workflow that uses Google Cloud Datastore."""
+"""A word-counting workflow that uses Google Cloud Datastore.
+
+This example shows how to use ``datastoreio`` to read from and write to
+Google Cloud Datastore. Note that running this example may incur charge for
```

Comment:
Yes, if you exceed the daily quota of read and writes. https://cloud.google.com/appengine/docs/quotas#Datastore